hack/testing/e2e_deployment.sh

```diff
@@ -0,0 +1,162 @@
+#!/bin/bash
+set -euo pipefail
+
+ORIGIN_APIMAN_ROOT=$(realpath --no-symlinks "$(dirname "$BASH_SOURCE")/../..")
+OS_ROOT=${OS_ROOT:-$(realpath --no-symlinks "$ORIGIN_APIMAN_ROOT/../origin")}
```

Comment:
I was trying to get rid of this requirement for having origin cloned nearby, or relying on it at all. It looks to me like the only need for OS_ROOT at this point is to include the openshift binaries in the PATH. Would it be unreasonable to just assume these will be in your path for this script?